.packit.yaml

```
@@ -0,0 +1,45 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: hammer_cli_foreman_leapp.gemspec
+
+# add or remove files that should be synced
+files_to_sync:
+    - hammer_cli_foreman_leapp.gemspec
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: hammer_cli_foreman_leapp
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-hammer_cli_foreman_leapp
+
+upstream_tag_template: "{version}"
```

`v{version}`

---

it looks like "{version}" in this [change](https://github.com/theforeman/hammer-cli-foreman-discovery/pull/56/commits/14f9dfe863b57bee952c80e6ee871a3bfc7262ca)

---

That's weird, we use `vX.Y.Z` format, see the [hammer-ansible](https://github.com/theforeman/hammer-cli-foreman-ansible/blob/master/.packit.yaml)

^^ @evgeni what's the correct approach?

---

Looking at https://github.com/theforeman/hammer-cli-foreman-leapp/tags this repository has prefixed tags with `v` while https://github.com/theforeman/hammer-cli-foreman-discovery/tags doesn't use that prefix.
```suggestion
upstream_tag_template: "v{version}"
```

---

Thanks, updated :)